compiler-rt/lib/rtsan/rtsan_interceptors.cpp

```diff
@@ -238,23 +243,35 @@ INTERCEPTOR(int, nanosleep, const struct timespec *rqtp,
 // Memory
 
 INTERCEPTOR(void *, calloc, SIZE_T num, SIZE_T size) {
+  if (rtsan_init_is_running && REAL(calloc) == nullptr)
```

Comment:
We should define `UseImpl` and use `DlsymAlloc::Use`. See #98415

Reply:
Fixed! Thanks